package.json

```diff
@@ -48,6 +48,7 @@
     "scrub": "node ./scripts/scrub.js",
     "start:legacy": "yarn workspace @uifabric/fabric-website-resources start",
     "start": "node scripts/start.js",
+    "start:docs": "yarn workspace @fluentui/docs start",
```

Comment:
since prompt menu is not accessible, this will allow blind users to start the experience

Reply:
don't you want to use `northstar` in the name?

Reply:
maybe we should then rename @fluentui/docs to @fluentui/northstar?

Reply:
but I agree - will rename the command here, let's rename remaining things in separate PR